modules/oc-mirror-mirror-to-mirror.adoc

```diff
@@ -70,7 +75,7 @@ ifdef::microshift[]
 * Convert the `ImageContentSourcePolicy` YAML content for use in manually configuring CRI-O.
 * If required, mirror the images from mirror to disk for disconnected or offline use.
 endif::microshift[]
-* Configure your cluster to use the resources generated by oc-mirror.
+* Configure your cluster to use the resources generated by oc-mirror. See _Configuring your cluster to use the resources generated by oc-mirror_.
 
 .Troubleshooting
```

Comment:
Technically out of scope of the PR, but I'll mention: A few lines below here, there's a period at the end of the link that could be removed. Also per https://redhat-documentation.github.io/supplementary-style-guide/#rh-kb-links suggest:

```
* link:https://access.redhat.com/solutions/7032017[Unable to retrieve source image] (Red Hat Knowledgebase)
```
or maybe more correctly:
```
* link:https://access.redhat.com/solutions/7032017[What to do when encountering "manifest unknown error: unable to retrieve source image" while using oc-mirror?] (Red Hat Knowledgebase)
```

Reply:
There's another instance like this in the "Mirroring from mirror-to-disk" section too.